### DISCUSSION TITLE
Absolute center mode when Stage Manager is on

### DISCUSSION THREAD

**Discussion by dangh** [rxhanson/Rectangle#880] [Ideas]
Hi,

I'm not sure about other users, but for me when I'm centering a window, it's because I want to read it without turning my head, so I prefer it to be centered absolutely on my screen.

It's maybe ridiculous to some people, but with the current center position when Stage Manager is on, it's still not center too me.

I would like to propose an absolute center mode, that will:
- Center the app relative to the screen, regardless of stage manager.
- If the window is overlap Stage Manager recent apps, we can move the window to the left/right to avoid overlapping, but still prefer the window to be close to the absolute center position.

WDYT?

**Comment by rxhanson** [comment]
I agree. I’ll put something in for this in the next release.
